pkg/cmd/bootstrap.go

```diff
@@ -120,6 +120,9 @@ func nonInteractiveMode(io *BootstrapParameters, client *utility.Client) error {
 			return fmt.Errorf("The mandatory flag %q has not been set", value)
 		}
 	}
+	if io.CommitStatusTracker && io.GitHostAccessToken == "" {
+		return fmt.Errorf("Kindly input the git-host-access-token if commit-status-tracker set to true")
```

Comment:
This doesn't need to be `Errorf` there's no format strings in the string?

Also, let's maybe tweak that to `--git-host-access-token is required when commit-status-tracker is enabled` ?